community_tasks/serbian_eval.py

```diff
@@ -0,0 +1,227 @@
+        suite=suite,
+        hf_repo=hf_repo,
+        hf_subset=hf_subset,
+        hf_avail_splits=hf_avail_splits,
+        evaluation_splits=evaluation_splits,
+        few_shots_split=few_shots_split,
+        few_shots_select="sequential",
+        metric=metric,
+        trust_dataset=True,
+        version=0,
+    )
+
+
+arc_easy_serbian_task = create_task_config(
+    task_name="serbian_evals:arc_easy",
+    prompt_function=serbian_eval_prompt,
+    hf_repo="datatab/serbian-llm-benchmark",
+    hf_subset=HFSubsets.ARC_EASY.value,
+    metric=[Metrics.loglikelihood_acc_norm],
+)
+
+arc_challenge_serbian_task = create_task_config(
+    task_name="serbian_evals:arc_challenge",
+    prompt_function=serbian_eval_prompt,
+    hf_repo="datatab/serbian-llm-benchmark",
+    hf_subset=HFSubsets.ARC_CHALLENGE.value,
+    metric=[Metrics.loglikelihood_acc_norm],
+)
+
+boolq_serbian_task = create_task_config(
+    task_name="serbian_evals:boolq",
+    prompt_function=serbian_eval_prompt,
+    hf_repo="datatab/serbian-llm-benchmark",
+    hf_subset=HFSubsets.BOOLQ.value,
+    metric=[Metrics.loglikelihood_acc_norm],
+)
+
+hellaswag_serbian_task = create_task_config(
+    task_name="serbian_evals:hellaswag",
+    prompt_function=serbian_eval_prompt,
+    hf_repo="datatab/serbian-llm-benchmark",
+    hf_subset=HFSubsets.HELLASWAG.value,
+    metric=[Metrics.loglikelihood_acc_norm],
+)
+
+openbook_qa_serbian_task = create_task_config(
+    task_name="serbian_evals:openbook",
+    prompt_function=serbian_eval_prompt,
+    hf_repo="datatab/serbian-llm-benchmark",
+    hf_subset=HFSubsets.OPENBOOK.value,
+    metric=[Metrics.loglikelihood_acc_norm],
+)
+
+piqa_serbian_task = create_task_config(
+    task_name="serbian_evals:piqa",
+    prompt_function=serbian_eval_prompt,
+    hf_repo="datatab/serbian-llm-benchmark",
+    hf_subset=HFSubsets.PIQA.value,
+    metric=[Metrics.loglikelihood_acc_norm],
+)
+
+oz_eval_task = create_task_config(
+    task_name="serbian_evals:oz_task",
+    prompt_function=serbian_eval_prompt,
+    hf_repo="datatab/serbian-llm-benchmark",
+    hf_subset=HFSubsets.OZ_EVAL.value,
+    metric=[Metrics.loglikelihood_acc],
+)
+
+winogrande_task = create_task_config(
+    task_name="serbian_evals:winogrande",
+    prompt_function=serbian_eval_prompt,
+    hf_repo="datatab/serbian-llm-benchmark",
+    hf_subset=HFSubsets.WINOGRANDE.value,
+    metric=[Metrics.loglikelihood_acc_norm],
+)
+
+
+TASKS_TABLE = [
+    arc_easy_serbian_task,
+    arc_challenge_serbian_task,
+    boolq_serbian_task,
+    hellaswag_serbian_task,
+    openbook_qa_serbian_task,
+    piqa_serbian_task,
+    oz_eval_task,
+    winogrande_task,
+]
+
+
+hello_message = """
+-----------------------------------
+------ Serbian LLM benchmark ------
+-----------------------------------
+
+Available tasks:
+-----------------
+{}
+"""
+task_names = "\n".join([t.name for t in TASKS_TABLE])
+hlog_warn(f"{hello_message.format(task_names)}")
```

Comment:
I would put this in the `if __name__ == "__main__":` as we do not want to print every tasks when this file is loaded